packages/client/src/pages/RecommendationsPage/RecommendationsTable/CustomPaginaton.tsx

```diff
@@ -0,0 +1,48 @@
+/*
+ * © 2021 Thoughtworks, Inc.
+ */
+
+import {
+  useGridSlotComponentProps,
+  GridToolbarContainer,
+} from '@material-ui/data-grid'
+import useStyles from './recommendationsTableStyles'
+import Pagination from '@material-ui/lab/Pagination'
+import { Box, MenuItem, Select, Typography } from '@material-ui/core'
+
+function CustomPagination(
+  handlePageSizeChange: (event: React.ChangeEvent<{ value: unknown }>) => void,
+) {
```

Comment:
You can add a return type for this component called `ReactElement` which is imported from `React`. 

```suggestion
function CustomPagination(
  handlePageSizeChange: (event: React.ChangeEvent<{ value: unknown }>) => void,
): ReactElement {
```